src/docs/release/breaking-changes/add-showAutocorrectionPromptRect.md

```diff
@@ -0,0 +1,90 @@
+---
+title: New method `showAutocorrectionPromptRect` added to `TextInputClient` 
+description: a new method `void showAutocorrectionPromptRect(int start, int end)` was added to the `TextInputClient` interface
```

Comment:
capitalization?

Reply:
(question because i don't know if this should be capitalized or not, we should check the other docs to be consistent)

Reply:
Yes, the description should start with "A", rather than "a". Also, we don't codefont a title. These aren't huge issues. The issue blocking me from merging is the release info.

Thanks, @LongCatIsLooong!